### PR TITLE
Added tlg0093.tlg003.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Theophrastus De sensu et sensibilibus

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |       +1 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +9052 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg003/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg003/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3960)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |       +1 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +9052 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg003/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg003/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3962)*

**Comment by AlisonBabeu** [comment]
hi @sonofmun both tlg0093.tlg003_cts_.xml and tlg0093.tlg004_cts_.xml need to have the language tag for the edition title description changed from "eng" to "mul"
```xml
 <ti:description xml:lang="eng">
```

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |       +1 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +9052 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg003/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg003/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3969)*

File: data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,1266 @@
...
+  <ref target="http://gallica.bnf.fr/ark:/12148/bpt6k28209z">Gallica</ref>
+ </biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+ <p>Text encoded in accordance with the latest EpiDoc standards</p>
+ <refsDecl n="CTS">
+  <cRefPattern matchPattern="(.+)" n="chapter"
+   replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
```
Comment (planatheisa) [inline data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml]: Wouldn't this get the fragment-div instead of the chapter-divs?

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +17 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +9052 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0093/tlg003/__cts__.xml`                   | New      |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | New      |
| `./data/tlg0093/tlg003/__cts__.xml`                   | Passing  |
| `./data/tlg0093/tlg003/tlg0093.tlg003.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/4007)*

**Pull request merged**